bindings/nodejs/lib/types/secret_manager/bridge/secret-manager.ts

```diff
@@ -70,3 +70,8 @@ export interface __SignSecp256k1EcdsaMethod__ {
 export interface __GetLedgerNanoStatusMethod__ {
     name: 'getLedgerNanoStatus';
 }
+
+export type __SetStrongholdPasswordMethod__ = {
```

Comment:
I don't know much TS but all the others are interfaces, can it be as well for consistency or does it really need to be a type?

Reply:
wew, fair point, I have no idea why I made it a type, changing this right away